tests/suites/test_suite_ssl.function

```diff
@@ -4675,12 +4675,27 @@ void tls13_srv_max_early_data_size(int scenario, int max_early_data_size_arg, in
             case TEST_EARLY_DATA_HRR:
                 ret = mbedtls_ssl_handshake(&(server_ep.ssl));
                 /*
-                 * Can be the case if max_early_data_size is smaller then the
-                 * smallest inner content or protected record.
+                 * In this write loop we try to always stay below the
+                 * max_early_data_size limit but if max_early_data_size is very
+                 * small we may exceed the max_early_data_size limit on the
+                 * first write. In TEST_EARLY_DATA_SERVER_REJECTS/
+                 * TEST_EARLY_DATA_HRR scenario, this is for sure the case if
+                 * max_early_data_size is smaller than the smallest possible
+                 * inner content/protected record. Take into account this
+                 * possibility here but only for max_early_data_size values
+                 * that are close to write_size. Below, chosen 1 for one byte
```

Comment:
I don't understand "chosen 1 for one byte of inner type and 16 bytes for AEAD expansion"

Does it mean `Below, '1' is for the inner type byte and '16' is for AEAD expansion (IV, ...)`?

If so, I still don't understand: surely IV isn't AEAD expansion, since CBC (for example) has an IV. AEADs uniquely have a tag (typically 16 bytes for AES-GCM), so is "AEAD expansion" the tag?

Reply:
Yes and Yes, fixing.